src/main/java/liquibase/ext/cassandra/lockservice/LockServiceCassandra.java

```diff
@@ -179,4 +179,12 @@ public boolean isDatabaseChangeLogLockTableInitialized(final boolean tableJustCr
         return isDatabaseChangeLogLockTableInitialized;
     }
 
+    private String getChangeLogTableName() {
```

Comment:
Hi. This method should be getChangeLogLockTableName where database.getDatabaseChangeLogLockTableName() statements are used. I tested it with the project using liquibase with custom names for changelog and changeloglock tables.

Reply:
I fixed that up, thanks